[Bug]: Unable to run executable lazygit

### Problem description

1. Update LunarVim from commit 02a1e769be9139dfe7294d24b7be2d09bb30b77f with `lvim +LvimUpdate +q` command.
2. Open LunarVim
3. See error log

### LunarVim version

Rolling branch

### Neovim version (>= 0.5)

NVIM v0.6.0

### Relevant log output

```shell
12:52:28 [ERROR] lvim: "Unable to run executable lazygit. Please make sure it is installed properly." file="terminal.lua", line=86
```


### Screenshots

_No response_

### I have

- [X] Read the readme
- [X] Read the wiki
- [X] Searched for similar issues

--- a/lua/lvim/core/terminal.lua
+++ b/lua/lvim/core/terminal.lua
@@ -83,7 +83,7 @@ end
 M.add_exec = function(opts)
   local binary = opts.cmd:match "(%S+)"
   if vim.fn.executable(binary) ~= 1 then
-    Log:error("Unable to run executable " .. binary .. ". Please make sure it is installed properly.")
+    Log:debug("Skipping configuring executable " .. binary .. ". Please make sure it is installed properly.")
     return
   end
 

